src/main/java/org/amahi/anywhere/server/ApiModule.java

```diff
@@ -46,10 +47,11 @@
 public class ApiModule {
     @Provides
     @Singleton
-    OkHttpClient provideHttpClient(ApiHeaders headers, HttpLoggingInterceptor logging) {
+    OkHttpClient provideHttpClient(ApiHeaders headers, HttpLoggingInterceptor logging, ChuckInterceptor chuck) {
         OkHttpClient.Builder clientBuilder = new OkHttpClient.Builder();
         clientBuilder.addInterceptor(headers);
         clientBuilder.addInterceptor(logging);
+        clientBuilder.addInterceptor(chuck);
```

Comment:
We can add a check here to add the interceptor only when we have to debug the api.